[Bug]: Could not build ballerina-lang repo when its path contains spaces

### Description

ballerina-lang repo master branch build failed with the following error message,
```
`> Task :jballerina-tools:generateDocs FAILED
Error: Could not find or load main class org.ballerinalang.plugin.gradle.doc.DocerinaGen
Caused by: java.lang.ClassNotFoundException: org.ballerinalang.plugin.gradle.doc.DocerinaGen

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':jballerina-tools:generateDocs'.
> Process 'command 'C:\Program Files\Eclipse Adoptium\jdk-11.0.20.8-hotspot\bin\java.exe'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.4/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 19s
346 actionable tasks: 41 executed, 305 up-to-date`
```
After relocating the repository to a path that does not include a space, the build was finished successfully.

### Steps to Reproduce

Clone the repository to a directory with that includes a space. Following is the path I copied it to,
` C:\Users\Thushara Piyasekara\Documents\Ballerina_Lang_Codebase\ballerina-lang>`

Run the following command in the clone repo directory,
`gradlew build -x check`


### Affected Version(s)

master branch

### OS, DB, other environment details and versions

OS: Windows 10

### Related area

-> CICD

### Related issue(s) (optional)

_No response_

### Suggested label(s) (optional)

_No response_

### Suggested assignee(s) (optional)

_No response_

--- a/distribution/zip/jballerina-tools/build.gradle
+++ b/distribution/zip/jballerina-tools/build.gradle
@@ -390,7 +390,7 @@ tasks.register('pathingJar', Jar) {
     doFirst {
         manifest {
             attributes "Class-Path": configurations.docerina.files.collect {
-                it.toURL().toString().replaceFirst("file:/", "/")
+                it.toURI().toString()
             }.join(' ')
         }
     }

